src/Microsoft.VisualStudio.Services.Agent/JobServerQueue.cs

```diff
@@ -123,7 +123,10 @@ public void Start(Pipelines.AgentJobRequestMessage jobRequest)
 
             if (jobRequest.Variables.TryGetValue(WellKnownDistributedTaskVariables.PostLinesSpeed, out var postLinesSpeed))
             {
-                Int32.TryParse(postLinesSpeed.Value, out _webConsoleLineUpdateRate);
+                if (!Int32.TryParse(postLinesSpeed.Value, out _webConsoleLineUpdateRate))
+                {
+                    _webConsoleLineUpdateRate = (int) _delayForWebConsoleLineDequeueDefault.TotalMilliseconds;
```

Comment:
This is just protectionary. If TryParse fails, I found out `_webConsoleLineUpdateRate` will be set to 0. Don't let that happen.